apps/volume-storage.html.md.erb

```diff
@@ -43,34 +27,52 @@ fly launch
 ...
 Wrote config file fly.toml
 ? Would you like to deploy now? No
+...
 Your app is ready! Deploy with `flyctl deploy`
 ```
 
+### Provision the volume
+
+Create a volume for the app, with the name you chose, in the same region you're deploying the app to. For example, the following command creates a volume called `myapp_data`, in region `lhr`, with 1GB of storage, for the app named `myapp`:
+
+```cmd
+fly volumes create myapp_data --region lhr --size 1 --app myapp
+```
+
```

Comment:
We don't have to do this part anymore, on initial launch, do we? It gets hairier when you want to add a volume where there was already a Machine, but that's the in-flux part you were talking about

Reply:
When or where do you create the volume to attach then?

Reply:
At first deployment, if there's a mounts section in fly.toml, I believe a new volume is created for the new machine that's also created. Once there are machines on the app, then you have to clone a machine that has a volume, or restart with no machines--but making this smoother is what @dangra is taking a stab at with https://github.com/superfly/flyctl/pull/2189, I believe.

Reply:
That is right, `fly deploy` will create a volume per process group that has a mount section defined in fly.toml **only on first deploy**.

And #2189 is about supporting the case of a user adding mounts after the first deploy. It won't create the volumes automatically, it will complain saying how many volumes are missing and ask the user to run `fly vol create` before attempting to deploy again.

Reply:
Testing this now.

Reply:
@dangra So is https://github.com/superfly/flyctl/pull/2189 the solution to adding a volume to an existing machine? You will need to create the volumes you need, update the fly.toml mounts section, and then run `fly deploy`? If you have, say, 4 machines in one process, will it add volumes to all of them (as long as 4 volumes are already created)?

Reply:
Yes, that is right.

Reply:
And I just merged it so you can update docs right away.

Reply:
I'm going to merge this PR and open a new one for the changes in https://github.com/superfly/flyctl/pull/2189.